Automattic/jetpack
# Block: Instagram - add animation to the image loading

Part of the fix for: #15277

#### Changes proposed in this Pull Request:
* Adds a fade between the image loading placeholder and the actual image to prevent some of the janky experience on initial load in the editor and when changing number of images displayed
* Add a notice to inform user if their account has less images that the number selected


#### Testing instructions:

- Checkout this PR to local Jetpack dev environment with beta blocks enabled
- Add an Instagram account and connect it, then play around with changing number of images, number of columns, etc. and saving and reloading
- Check that there is a fade transitions between loading placeholders and actual image

**Before:**
![before-ani](https://user-images.githubusercontent.com/3629020/78741295-11040280-79ad-11ea-9e9b-72bb831fe274.gif)

**After:**
![2020-04-14 11 58 21](https://user-images.githubusercontent.com/3629020/79171898-5447f080-7e47-11ea-82fd-ac14d62ddd83.gif)

#### Proposed changelog entry for your changes:
* No changelog entry needed
